dimod/reference/composites/connectedcomponent.py

```diff
@@ -94,9 +94,10 @@ def sample(self, bqm, components=None, **parameters):
         if isinstance(components, set):
             components = [components]
         sampleset = None
+        fixed_value = list(bqm.vartype.value)[0]
```

Comment:
`bqm.vartype.value` is actually a FrozenSet which is unordered, this should be `fixed_value = min(bqm.vartype.value)`